aws-ssh-config.py

```diff
@@ -150,6 +152,9 @@ def main():
             hostid = args.prefix + instance_id
             hostid = hostid.replace(' ', '_') # get rid of spaces
 
+            if args.postfix:
+                hostid + '_' + args.postfix
```

Comment:
Would it be possible to treat `args.postfix` like `args.prefix` and just concatenate it without `_` on line 152?